qtpy/QtGui.py

```diff
@@ -14,6 +14,24 @@
 
 _missing_optional_names = {}
 
+_QTOPENGL_NAMES = [
+    'QOpenGLBuffer',
+    'QOpenGLFramebufferObject',
+    'QOpenGLFramebufferObjectFormat',
+    'QOpenGLShader',
+    'QOpenGLShaderProgram',
+    'QOpenGLContext',
+    'QOpenGLContextGroup',
+    'QOpenGLDebugLogger',
+    'QOpenGLDebugMessage',
+    'QOpenGLPixelTransferOptions',
+    'QOpenGLTexture',
+    'QOpenGLTextureBlitter',
+    'QOpenGLVersionProfile',
+    'QOpenGLVertexArrayObject',
+    'QOpenGLWindow',
+]
```

Comment:
```suggestion
_QTOPENGL_NAMES = {
    'QOpenGLBuffer',
    'QOpenGLFramebufferObject',
    'QOpenGLFramebufferObjectFormat',
    'QOpenGLShader',
    'QOpenGLShaderProgram',
    'QOpenGLContext',
    'QOpenGLContextGroup',
    'QOpenGLDebugLogger',
    'QOpenGLDebugMessage',
    'QOpenGLPixelTransferOptions',
    'QOpenGLTexture',
    'QOpenGLTextureBlitter',
    'QOpenGLVersionProfile',
    'QOpenGLVertexArrayObject',
    'QOpenGLWindow',
}
```

Also, tiny nit but semantically and efficiently this makes most sense as a set.

Reply:
Works for me.